### ISSUE TITLE
Don't report *_test packages

### ISSUE THREAD

**Issue by cloudlena** [kyoh86/scopelint#2]
I usually declare my tests in separate `*_test` packages. This means I have the following files:

```txt
internal/mypkg/mypkg.go (uses package `mypkg`
internal/mypkg/mypkg_test.go (uses package `mypkg_test` to test the publicly exposed API from `mypkg`
```

`scopelint` returns an error in that project:

```txt
internal/mypkg/mypkg_test.go is in package mypkg_test, not mypkg
```

However, the above setup is perfectly valid and even recommended.
